lib/tasks/index.rake

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'arclight'
+
+# hardcoded since we don't have access to Blacklight.connection_config[:url] here
+SOLR_URL = ENV['SOLR_URL'] || 'http://127.0.0.1:8983/solr/blacklight-core'
```

Comment:
I wonder if this would be better written as `ENV['SOLR_URL'] ||= 'http...` and then not set a local `SOLR_URL` constant at all.

If we did that we could kill line 29 in this file as well I think.